live-editing/configs/LocalizationConfigGenerator.ts

```diff
@@ -5,62 +5,57 @@ import {
     IgxTimePickerModule,
     IgxSelectModule
 } from 'igniteui-angular';
-import {AppModuleConfig, Config, IConfigGenerator} from 'igniteui-live-editing'
+import {AppModuleConfig, Config, IConfigGenerator} from 'igniteui-live-editing';
 export class LocalizationConfigGenerator implements IConfigGenerator {
     public generateConfigs(): Config[] {
         const configs = new Array<Config>();
 
         // localization sample 1
         configs.push(new Config({
             component: 'LocalizationSample1Component',
-            additionalFiles: ["src/app/data/nwindData.ts"],
-            additionalDependencies: ["igniteui-angular-i18n"],
+            additionalFiles: ['/src/app/data/nwindData.ts'],
+            additionalDependencies: ['igniteui-angular-i18n'],
             appModuleConfig: new AppModuleConfig({
                 imports: ['IgxInputGroupModule', 'LocalizationSample1Component', 'IgxGridModule', 'IgxIconModule'],
                 ngDeclarations: ['LocalizationSample1Component'],
                 ngImports: ['IgxInputGroupModule', 'IgxGridModule', 'IgxGridModule']
-            }),
-            shortenComponentPathBy: "/localization-samples/"
+            })
         }));
 
         // localization sample 2
         configs.push(new Config({
             component: 'LocalizationSample2Component',
-            additionalFiles: ["src/app/data/nwindData.ts"],
-            additionalDependencies: ["igniteui-angular-i18n"],
+            additionalFiles: ['/src/app/data/nwindData.ts'],
+            additionalDependencies: ['igniteui-angular-i18n'],
             appModuleConfig: new AppModuleConfig({
                 imports: ['IgxInputGroupModule', 'LocalizationSample2Component', 'IgxIconModule',
                           'IgxTimePickerModule', 'IgxGridModule'],
                 ngDeclarations: ['LocalizationSample2Component'],
                 ngImports: ['IgxInputGroupModule', 'IgxIconModule', 'IgxTimePickerModule', 'IgxGridModule']
-            }),
-            shortenComponentPathBy: "/localization-samples/"
+            })
         }));
 
         // localization sample 3
         configs.push(new Config({
             component: 'LocalizationSample3Component',
-            additionalFiles: ["src/app/data/nwindData.ts"],
-            additionalDependencies: ["igniteui-angular-i18n"],
+            additionalDependencies: ['igniteui-angular-i18n'],
```

Comment:
additionalFiles should not be removed as the sample in stackblitz will return an error. Is there a reason to remove adding the nwindData.ts?